Station/pycpt-seasonal-station-grid-Chile.ipynb

```diff
@@ -0,0 +1,794 @@
+    "})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "076ded6b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MOS = 'CCA' # must be one of 'CCA', 'PCR'\n",
+    "\n",
+    "# Use dl.observations.keys() to see all options for predictand \n",
+    "# and dl.hindcasts.keys() to see all options for predictors.\n",
+    "# Make sure your first_year & final_year are compatible with \n",
+    "# your selections for your predictors and predictands.\n",
+    "predictor_names = [ \"CanSIPSIC3.PRCP\", 'SEAS51.PRCP']\n",
+    "#predictor_names = [\"GEOSS2S.TMIN\"]\n",
+    "\n",
+    "\n",
+    "# To download predictand observations from the Data Library, set\n",
+    "#\n",
+    "#   local_predictand_file = None\n",
+    "#\n",
+    "# To read predictand data from a local file instead,\n",
+    "# set local_predictand_file to the full pathname of the file. e.g.\n",
+    "#\n",
+    "#   local_predictand_file = \"/home/aaron/src/pycpt_notebooks/obs_PRCP_Oct-Dec.tsv\"\n",
+    "#\n",
+    "# The file should be formatted according to the following guidelines:\n",
+    "# https://cpthelp.iri.columbia.edu/CPT_use_input_gridded.html\n",
+    "# https://cpthelp.iri.columbia.edu/CPT_use_input_stations.html\n",
+    "\n",
+    "# TODO applying the predictand extent constraints causes a segfault in ingrid.\n",
+    "#predictand_name = 'CHILESTATIONS_2020.TMIN'\n",
+    "predictand_name = 'CHILESTATIONS_2020.PRCP'\n",
+    "#\n",
+    "#predictand_name = 'CR2MET0p25.TMIN'\n",
+    "#predictand_name = 'CR2MET0p25.PRCP'\n",
+    "use_dlauth = True\n",
+    "local_predictand_file = None\n",
+    "\n",
+    "    \n",
+    "download_args = { \n",
+    "    # 'fdate':\n",
+    "    #   The initialization date of the model forecasts / hindcasts.\n",
+    "    #   This field is defined by a python datetime.datetime object,\n",
+    "    #   for example: dt.datetime(2022, 5, 1) # YYYY, MM, DD as integers\n",
+    "    #   The year field is only used for forecasts, otherwise ignored.\n",
+    "    #   The day field is only used in subseasonal forecasts, otherwise ignored.\n",
+    "    #   The month field is an integer representing a month - ie, May=5.\n",
+    "    'fdate':  dt.datetime(2023, 5, 1),\n",
+    "\n",
+    "    # 'first_year':\n",
+    "    #   The first year of hindcasts you want. **NOT ALL MODELS HAVE ALL YEARS**.\n",
+    "    #   Double check that your model has hindcast data for all years in [first_year, final_year].\n",
+    "    #   This field is defined by a python integer representing a year, e.g. 1993.\n",
+    "    'first_year': 1981,  \n",
+    "\n",
+    "    # 'final_year':\n",
+    "    #   The final year of hindcasts you want. **NOT ALL MODELS HAVE ALL YEARS**.\n",
+    "    #   Double check that your model has hindcast data for all years in [first_year, final_year].\n",
+    "    #   This field is defined by a python integer representing a year, e.g. 2016.\n",
```

Comment:
Using `target_first_year` and `target_final_year` instead of `first_year` and `final_year` makes it easier to automate configurations like we use in FbF, where there is a fixed target season and we generate forecasts at multiple leads for that season. `first_year` and `final_year` still work, but I would only use them in situations where you plan to generate year-round rolling starts. If that's the case, or if you don't know yet, this is fine.

Reply:
> Using `target_first_year` and `target_final_year` instead of `first_year` and `final_year` makes it easier to automate configurations like we use in FbF, where there is a fixed target season and we generate forecasts at multiple leads for that season. `first_year` and `final_year` still work, but I would only use them in situations where you plan to generate year-round rolling starts. If that's the case, or if you don't know yet, this is fine.

In this case, DMC has indicated that they will do 'rolling starts'